boot/bootdata/bootcd.ini

```diff
@@ -1,6 +1,6 @@
 [FREELOADER]
 DefaultOS=Setup
```

Comment:
Shouldn't default choice be `Setup_Debug` to avoid changing the current behavior? Or it's overridden by `txtsetup.sif` by default anyway?

Reply:
Actually it's currently already done by txtsetup.sif, because both the OsLoadOptions + SetupDebugOptions values are combined together automatically, due to the fact that do that systematically when compiling freeldr in DBG mode.
But there is also a difference: the SetupDebugOptions currently active use /KDSERIAL, which is nice for the test bots, but not when you want to interact yourself with the debugger on real hardware, iirc.

Reply:
There is also no interactivity when ReactOS boot from bootcd in Virtual Box (5.1.30-32, XP Host).
If "Options = / NOGUIBOOT / DEBUG / DEBUGPORT = COM1 / BAUDRATE = 115200 / FIRSTCHANCE" is used then I can debug bootcd.

Reply:
When using DEBUGPORT=COMx, the /KDSERIAL means that the debugger input should come from the host computer (that receives the data from serial for display) instead of the target computer being debugged.